src/pages/SaibaMais/index.tsx

```diff
@@ -1,12 +1,12 @@
 import { LetsMake } from "../../components/LetsMake";
 import Cards from "../../components/ContainerFuncionalidadeRequisitos";
-import { CardInformativo } from "../../components/CardInformativo";
+// import { CardInformativo } from "../../components/CardInformativo";
 export const SaibaMaisPage = () => {
   return (
     <div>
       <LetsMake />
       <Cards />
-      <CardInformativo />
+      {/* <CardInformativo /> */}
```

Comment:
Seria bom conversamos sobre essa remoção com o time, pois é um conteúdo que faz parte da página Saiba Mais. Pelo que vi no Jira, não há história para refatoração do Saiba Mais, então é um conteúdo que está sendo perdido se for para produção. Talvez seja interessante uma história para refatorar essa página, ou pelo menos uma task para adicionar o CardInformativo no novo formato que o componente espera.